checker/checkers.go

```diff
@@ -114,13 +116,25 @@ type typeAndPriority struct {
 }
 
 func determineTypeAndPriorityForPaths(assetPaths []string) (map[string]typeAndPriority, error) {
-	typeAndPriorities := make(map[string]typeAndPriority)
-	for _, assetPath := range assetPaths {
-		typeAndPriorityForAsset, err := determineTypeAndPriority(assetPath)
-		if err != nil {
-			return nil, err
-		}
-		typeAndPriorities[assetPath] = typeAndPriorityForAsset
+	typeAndPriorities := map[string]typeAndPriority{}
+	var mapLock sync.Mutex
+	g := errgroup.Group{}
```

Comment:
nit: prefer var-style init rather than empty struct literal (although don't feel as strongly about this as `make(map)` style)

```suggestion
var (
	mapLock sync.Mutex
	g errgroup.Group
)
```

Reply:
Done